app.js

```diff
@@ -1,8 +1,17 @@
 // import functions and grab DOM elements
+const nameInput = document.getElementById('name-input');
+const changeButton = document.getElementById('change-name');
+const nameDisplay = document.getElementById('name-display');
+console.log(nameDisplay);
+
+changeButton.addEventListener('click', () => {
+    console.log('clicking');
```

Comment:
and you've got some functionality on your button (and using a console.log to confirm it's working)